src/sass/abstracts/_variables.scss

```diff
@@ -0,0 +1,17 @@
+// COLOR VARIABLES
+:root {
+  --color-primary: #228be6; // Blue 6 -Open Colors
+  --color-primary-dark: #1864ab; // Blue 9 -Open Colors
+  --color-primary-light: #4dabf7; // Blue 4 -Open Colors
+
+  --color-secondary: #e67d22; // Primary's Contrast
+
+  --color-gray-dark: #374151; // Gray 700 -Tailwind
+  --color-gray-light: #9ca3af; // Gray 400 -Tailwind
+
+  --color-white: #eff6ff; // Blue 50 -Tailwind
+  --color-white-dark: #dbeafe; // Blue 100 -Tailwind
```

Comment:
why are you calling a blue color white?

Reply:
what the use case of this color?

Reply:
Read about what semantic colors mean

https://dev.to/ynab/a-semantic-color-system-the-theory-hk7

Reply:
> why are you calling a blue color white?

The colors resemble more towards white, with a small tint of blue. I am using them wherever I need a white color so it is in flow with the primary color.

Reply:
> what the use case of this color?

The color is associated with calmness and a professional feel, which I thought is appropriate for this application as books can be related to both.

But after making the homepage I think it might not be the right choice, will look more into it.